### PR TITLE
EKS Prep

### PR DESCRIPTION
**Pull request by yakloinsteak**
## Description

These are changes that should:
1. Not break our ECS deployments
2. Make things work or work better on kubernetes

## Type of change
- [x] Code clean-up / housekeeping

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: lib/rds_sql_server/rds.rb [hunk 1]
```diff
@@ -37,19 +35,7 @@ def self.timeout
...
-                                           region: REGION,
-                                           access_key_id: ACCESS_KEY_ID,
-                                           secret_access_key: SECRET_ACCESS_KEY,
-                                         })
-    else
-      self.client = Aws::RDS::Client.new({
-                                           region: REGION,
-                                         })
-    end
+    self.client = Aws::RDS::Client.new
```
Comment (yakloinsteak) [inline lib/rds_sql_server/rds.rb:38]: We should abandon long-lived keys, and spot checking a few roles in ECS, it looks like these keys aren't serving a purpose. Let me know if there's some reason to keep the ability to inject different static creds into this code or if I'm mistaken.

Comment (eanders) [reply lib/rds_sql_server/rds.rb:38]: Hmm, we might still need the region (see line 50).  I'm pretty sure we fixed up the roles, but we should definitely test that.

**Comment by yakloinsteak** [comment]
@eanders Could I deploy this to QA on ECS? Is release-121 okay? I'll need help with the LSA/RDS QA, though.

**Comment by yakloinsteak** [comment]
All seems well in the staging environment.

**Pull request merged**